dotnet/nuget/nuget-package.props

```
@@ -1,15 +1,15 @@
 <Project>
   <PropertyGroup>
     <!-- Central version prefix - applies to all nuget packages. -->
-    <VersionPrefix>1.46.0</VersionPrefix>
+    <VersionPrefix>1.47.0</VersionPrefix>
```

Some overlap of changes in the PR.

```suggestion
    <VersionPrefix>1.47.0</VersionPrefix>
```